keel-clouddriver/keel-clouddriver.gradle.kts

```diff
@@ -12,7 +12,6 @@ dependencies {
   implementation("com.fasterxml.jackson.module:jackson-module-kotlin")
   implementation("com.github.ben-manes.caffeine:caffeine")
   implementation("com.fasterxml.jackson.datatype:jackson-datatype-jsr310")
-  implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core")
   implementation("org.jetbrains.kotlinx:kotlinx-coroutines-jdk8")
```

Comment:
`kotlinx-coroutines-jdk8` transitively pulls in `kotlinx-coroutines-core` so there's no need to specify both